src/component/contents/DraftEditorBlock.react.js

```diff
@@ -204,6 +213,7 @@ class DraftEditorBlock extends React.Component {
     return (
       <div data-offset-key={offsetKey} className={className}>
         {this._renderChildren()}
```

Comment:
Sounds good to me. Maybe we should even abstract both of them into a single facade. We could name the facade _renderChildren, and rename the current '_renderChildren' to something else. This way we could do the logic of checking if a blockType is a valid container in there.